app/serializers/claim_base_serializer.rb

```diff
@@ -1,5 +1,33 @@
 # frozen_string_literal: true
 class ClaimBaseSerializer < ActiveModel::Serializer
-  attributes :id, :date_filed, :min_est_date, :max_est_date,
+  attributes :id, :evss_id, :date_filed, :min_est_date, :max_est_date,
              :tracked_items, :phase_dates, :open, :waiver_submitted
+
+  def date_filed
+    object.data['date']
```

Comment:
We could do the mapping in an `after_initialize` hook to set the attributes on the claim model directly, but I think that the serializer is a more appropriate place for this logic as the attributes are only used for presentation. I considered doing some metaprogramming to define these methods but opted for the more explicit definitions as its easier to read and comprehend